variables.tf

```
@@ -182,30 +192,28 @@ variable "watson_discovery_service_endpoints" {
 
 variable "existing_assistant_instance" {
   default     = null
-  description = "CRN of the an existing watsonx Assistance instance."
+  description = "CRN of an existing watsonx Assistance instance."
   type        = string
 }
 
 variable "watsonx_assistant_plan" {
   default     = "do not install"
-  description = "The plan that's used to provision the watsonx Assistance instance."
+  description = "The plan that is used to provision the watsonx Assistance instance."
```

```suggestion
  description = "The plan that is used to provision the watsonx Assistant instance."
```

Please remove the `Assistance` reference. The service name is `watsonx Assistant`

---

Addressed.